WcaOnRails/db/migrate/20161201211133_add_additional_fields_for_delegates.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+class AddAdditionalFieldsForDelegates < ActiveRecord::Migration
+  def change
+    add_column :users, :location_description, :string
```

Comment:
OK, "location" usually means lat/long in this system.

I probably prefer `location_text`, assuming it's a free form text about where the delegate lives.

Just a suggestion. By all means, keep this if you like it.